dev/partkey.md

```diff
@@ -172,3 +172,41 @@ wrapped in a $\Credential$ struct, containing the following fields:
 
 And this verified credential is wrapped in a $\Vote$ struct with _Raw Vote_ 
 ($\mathrm{R}$), _Verified Credential_ ($\Credential$), and _Signature_ ($\Sig$).
+
+
+## Algorand State Proof Keys 
+### Algorand's Committable Ephemeral Keys Scheme - Merkle Keystore
+
+Algorand archives [forward security](https://en.wikipedia.org/wiki/Forward_secrecy) using the Merkle Keystore scheme. This scheme consists of designate an ephemeral key for each round in which it will be used. The scheme uses a Merkle tree to generate a commitment on those keys. 
```

Comment:
```suggestion
Algorand achieves [forward security](https://en.wikipedia.org/wiki/Forward_secrecy) using the Merkle Keystore scheme. This scheme consists in using a different ephemeral key for each round in which it will be used. The scheme uses a Merkle tree to generate a commitment on those keys. 
```

Reply:
it says "round" here, but "participation period" later.

My understanding is that "participation period" is actually longer than rounds. Can you please define "participation period" and make everything consistent?